src/PerformanceTests/Common/Scenarios/BaseRunner.cs

```diff
@@ -109,17 +124,33 @@ ISendOnlyBus CreateSendOnlyEndpoint()
     IBus CreateEndpoint()
     {
         var configuration = CreateConfiguration();
-        configuration.PurgeOnStartup(false);
         configuration.EnableFeature<NServiceBus.Performance.SimpleStatisticsFeature>();
         configuration.CustomConfigurationSource(this);
+
+        if (QueuesWerePurgedWhenSeedingData())
+            configuration.PurgeOnStartup(false);
+        else
+            configuration.PurgeOnStartup(true);
+
         return Bus.Create(configuration).Start();
     }
 
-    Configuration CreateConfiguration()
+    private bool QueuesWerePurgedWhenSeedingData()
+    {
+        if (this is ICreateSeedData) return true;
+        return false;
+    }
+
+    BusConfiguration CreateConfiguration()
     {
         var configuration = new Configuration();
         configuration.EndpointName(EndpointName);
         configuration.EnableInstallers();
+        configuration.DiscardFailedMessagesInsteadOfSendingToErrorQueue();
```

Comment:
Don't remove `DiscardFailedMessagesInsteadOfSendingToErrorQueue`. This should not be disabled for *all* test.